nautobot/docs/rest-api/authentication.md

```diff
@@ -11,7 +11,7 @@ The Nautobot REST API primarily employs token-based authentication. For convenie
 
 An authentication token is attached to a request by setting the `Authorization` header to the string `Token` followed by a space and the user's token:
 
-```
+```bash
 $ curl -H "Authorization: Token $TOKEN" \
 -H "Accept: application/json; indent=4" \
 http://nautobot/api/dcim/sites/
```

Comment:
Here and in the example below, possibly split the code block into a `bash` block containing the `curl` command and a `json` block containing the response?

Reply:
Sure I did this and split it into 2 code blocks.
In looking this up, I found `console`, which also works just like bash but dims/colorizes the `$` in the beginning of the line.  We can use that too but it's so minor I'm not sure if it matters.
Leaving it as `bash` for now.